pyqtgraph/graphicsItems/TextItem.py

```diff
@@ -66,6 +68,7 @@ def setText(self, text, color=None):
         This method sets the plain text of the item; see also setHtml().
         """
         if color is not None:
+            self.color = color
```

Comment:
if `setColor` is going to be a method, then `color` should not be a property, but be a getter (to line up with the Qt API more).  Property can probably be `self._color`?